### ISSUE TITLE
missing doc for torch.nn.functional.group_norm

### ISSUE THREAD

**Issue by sytrus-in-github** [pytorch/pytorch#34209]
## 📚 Documentation

When browsing through the docs on normalization layers, I realized that `torch.nn.functional.group_norm` is missing from the documentation (the `torch.nn.GroupNorm` layer counterpart is there). I am wondering why it is missing. My guess would be that this is not the intended behaviour.
